src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkTests.ps1

```diff
@@ -187,6 +187,95 @@ function Test-subnetCRUD
     }
 }
 
+<#
+.SYNOPSIS
+Tests creating new virtualNetwork w/ delegated subnets.
+.DESCRIPTION
+SmokeTest
+#>
+function Test-subnetDelegationCRUD
```

Comment:
you need to add the test recodings for the tests to both git and the project. Example [here](https://github.com/Azure/azure-powershell/tree/preview/src/ResourceManager/Network/Commands.Network.Test/SessionRecords/Commands.Network.Test.ScenarioTests.VirtualNetworkTests) and here](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj#L321).

Reply:
static analysis errors should be fixed in the latest build.
Running a local build to re-run tests before I add the recordings. Should be there soon.

Reply:
I added these cmdlets to the exception, but I'm still experiencing build failures :(